crates/fuel-core/src/state/historical_rocksdb.rs

```diff
@@ -384,6 +429,21 @@ where
             Ok(())
         }
     }
+
+    pub fn v1_entries(&self) -> BoxedIter<StorageResult<(u64, Changes)>> {
+        self.db
+            .iter_all::<ModificationsHistoryV1<Description>>(None)
+    }
+
+    // Signals that the migration is complete
+    pub fn complete_migration(&self) {
+        match self.migration_in_progress.write() {
+            Err(poisoned) => tracing::warn!(
+                "The lock guarding the migration status is poisoned: {poisoned:?}"
+            ),
+            Ok(mut write_guard) => *write_guard = true,
```

Comment:
```suggestion
            Ok(mut write_guard) => *write_guard = false,
```

Reply:
Good catch (and thanks :D). Changed in 14516d17e9